### ISSUE TITLE
Virtual card monthly limit bugs

### ISSUE THREAD

**Issue by marcogbarcellos** [opencollective/opencollective#1346]
- [x] This is the most serious one: The virtual card query is considering the donation per currency. so if you have a U$50 virtual card and you spend U$50 you'll still be able to spend €50 because it's a different currency.
- [x] Email 0 -> It's considering only the `amount` field and not the `monthlyLimitPerMember`. Also images broken.
![0email](https://user-images.githubusercontent.com/8717041/46639027-d4deb000-cb31-11e8-8ca9-b7a869eaabeb.png)

- [x] Redeem Page not working properly because of the same problem as above(`amount` only, and not counting the `monthlyLimitPerMember` case.
![redeem0](https://user-images.githubusercontent.com/8717041/46639125-461e6300-cb32-11e8-8a7e-994d56e93ec1.png)

- [x] After a virtual card balance reaches $0, it should not be shown in the payment method options anymore. In the following print, one of my $50 virtual card has not balance anymore so it should be filtered and not be there.
![screen shot 2018-10-08 at 4 31 54 pm](https://user-images.githubusercontent.com/8717041/46641329-145fc900-cb3f-11e8-8dc4-a6f55e356ca6.png)

- [x] @piamancini's virtual card(monthly card) is not showing up when it should be showing up.

**Comment by marcogbarcellos** [comment]
Regarding the last issue, I was able to reproduce in staging. I tested something to make sure it's something on either the payment method options query or UI.

These are the ones that are showing up in staging.
![screen shot 2018-10-09 at 12 16 55 pm](https://user-images.githubusercontent.com/8717041/46683219-a1496780-cbbd-11e8-8b8c-d59abcc20488.png)

To test and see if the monthly card would show up, I've marked all other payment methods with the`deletedAt` field, leaving only the monthly payment. It finally shows up so it means that there's some filter either on the frontend or the backend that's limiting the amount of payment methods to be shown on the box:
![screen shot 2018-10-09 at 12 14 04 pm](https://user-images.githubusercontent.com/8717041/46683290-d81f7d80-cbbd-11e8-9f82-ca4f11446c1c.png)

Will check why now...
